code/controller/actuator.cpp

```diff
@@ -22,152 +23,148 @@ Actuator::Actuator(const QString& serial_port,
+
+    return result;
 }
 
 void Actuator::resetDeviceNumber() {
-	char instr[] = { 0, ZaberCmd::RENUMBER, 0, 0, 0, 0 };
-	
-	m_serial_port->write(instr);
+    char instr[] = { 0, ZaberCmd::RENUMBER, 0, 0, 0, 0 };
+
+    m_serial_port->write(instr);
 
-	m_x_device = 0;
-	m_y_device = 1;
+    m_x_device = 0;
+    m_y_device = 1;
 }
 
 int const Actuator::intPow(int x, int p) {
-	if (p == 0) return 1;
-	if (p == 1) return x;
+    if (p == 0) return 1;
+    if (p == 1) return x;
 
-	int tmp = intPow(x, p / 2);
-	if (p % 2 == 0) return tmp * tmp;
-	else return x * tmp * tmp;
+    int tmp = intPow(x, p / 2);
+    if (p % 2 == 0) return tmp * tmp;
+    else return x * tmp * tmp;
 }
 
 int Actuator::setSerPort(const QString& serial_port) {
-	if (m_serial_port->portName() == serial_port) {
-		return 0;
-	}
-	//close the current port
-	if (m_serial_port->isOpen()) {
-		m_serial_port->flush();
-		m_serial_port->close();
-	}
-	
-	m_serial_port->setPortName(serial_port);
-	if (m_serial_port->lastError() != 0) {
-		Logger::log(m_serial_port->errorString().toStdString(), Logger::ERROR);
-		return -1;
-	}
-	return 0;
+    if (m_serial_port->portName() == serial_port) {
+        return 0;
+    }
+    //close the current port
+    if (m_serial_port->isOpen()) {
+        m_serial_port->flush();
+        m_serial_port->close();
+    }
+
+    m_serial_port->setPortName(serial_port);
+    if (m_serial_port->lastError() != 0) {
+        Logger::log(m_serial_port->errorString().toStdString(), Logger::ERROR);
+        return -1;
+    }
+    return 0;
 }
 
 int Actuator::changeSettings(const PortSettings& settings) {
-	m_serial_port->setBaudRate(settings.BaudRate);
-	m_serial_port->setDataBits(settings.DataBits);
-	m_serial_port->setParity(settings.Parity);
-	m_serial_port->setStopBits(settings.StopBits);
-	m_serial_port->setFlowControl(settings.FlowControl);
-	m_serial_port->setTimeout(settings.Timeout_Millisec);
-
-	if (m_serial_port->lastError() != 0) {
-		Logger::log(m_serial_port->errorString().toStdString(), Logger::ERROR);
-		return -1;
-	}
-	return 0;
+    m_serial_port->setBaudRate(settings.BaudRate);
+    m_serial_port->setDataBits(settings.DataBits);
+    m_serial_port->setParity(settings.Parity);
+    m_serial_port->setStopBits(settings.StopBits);
+    m_serial_port->setFlowControl(settings.FlowControl);
+    m_serial_port->setTimeout(settings.Timeout_Millisec);
+
+    if (m_serial_port->lastError() != 0) {
+        Logger::log(m_serial_port->errorString().toStdString(), Logger::ERROR);
+        return -1;
+    }
+    return 0;
 }
 
 void Actuator::switchDevices() {
-	unsigned char temp = m_x_device;
-	m_x_device = m_y_device;
-	m_y_device = temp;
+    unsigned char temp = m_x_device;
+    m_x_device = m_y_device;
+    m_y_device = temp;
 }
 
 Actuator::~Actuator() {
     delete m_serial_port;
 }
 
-void Actuator::move(Dir dir, int time) {
-	move(Controller::toVec2(dir), time);
+void Actuator::move(Dir dir, int t) {
+    move(Controller::toVec2(dir), t);

```

Comment:
Why exactly did you change it? Very poor choice for variable name, remember, for code to be maintainable and in good taste, it should be readable, maintainable and the purpose as obvious as can be at first look.